code/ai/aicode.cpp

```diff
@@ -6998,10 +6998,24 @@ void attack_set_accel(ai_info *aip, ship_info *sip, float dist_to_enemy, float d
 		return;
 	}
 
+	// assume an optimal range of 200m
+	float optimal_range = 200.0f;
```

Comment:
We should use a `constexpr` or `#define` to avoid magic numbers and so that this can be referenced elsewhere.